c7n/query.py

```diff
@@ -746,7 +747,14 @@ def _scalar_augment(manager, model, detail_spec, client, resource_set):
     results = []
     for r in resource_set:
         kw = {param_name: param_key and r[param_key] or r}
-        response = op(*args, **kw)
+        try:
+            response = op(*args, **kw)
+        except ClientError as e:
+            if (e.response['Error']['Code'] in ecodes_acc_denied and
+            e.response['ResponseMetadata']['HTTPStatusCode'] == 403 and
+            'resource-based policy' in e.response['Error']['Message']):
+                manager.log.exception(e)
+                continue
```

Comment:
Would it make sense to raise for all other errors? My assumption is that if we leave it as is, the boto3 errors will bubble up automatically